rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1254,6 +1254,9 @@ private void processReleaseTransactions(Transaction rskTx) {
             else {
                 txsWaitingForSignatures.put(rskTx.getHash(), entry.getTransaction());
             }
+            if(activations.isActive(ConsensusRule.RSKIP326)) {
+                eventLogger.logPegoutConfirmed(entry.getTransaction().getHash(), rskExecutionBlock.getNumber());
```

Comment:
According to the RSKIP the block number in the event should be the height where the pegout was created, right? 

What you are passing here is the current height, so it would be the height where the pegout is moved to waiting for signatures. Which is basically the same height where the event is emitted